roles/collect/defaults/main.yml

```
@@ -4,6 +4,8 @@ collect_format: 'csv'
 collect_manifest_uuid: '{{ 99999999999999999999 | random | to_uuid }}'
 collect_archive_name: cost-mgmt
 ocp_validate_cert: 'true'
+ocp_cluster_id: ''
+token_name: ''
```

Can we be more explicit with the name?
```suggestion
reporting_operator_token_name: ''
```